t/venom/myapp/test.yml

```diff
@@ -0,0 +1,11 @@
+name: Title of TestSuite
+testcases:
+
+  - name: Load database fixtures
+    steps:
+      - type: dbfixtures
+        database: mysql
+        dsn: root:inverse@(172.20.120.7:3306)/pf
+        folder: fixtures
+        files:
+          - password.yml
```

Comment:
As mentioned in the [Venom's docs](https://github.com/ovh/venom/tree/master/executors/dbfixtures#input), `files` is a fallback when `folder` is not used. From my point of view, we should only use `folder`.